streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -76,10 +79,10 @@
 
 @SuppressWarnings("unchecked")
 public class StreamsPartitionAssignorTest {
-    private final String c1 = "consumer1";
-    private final String c2 = "consumer2";
-    private final String c3 = "consumer3";
-    private final String c4 = "consumer4";
+    private static final String CONSUMER_1 = "consumer1";
+    private static final String CONSUMER_2 = "consumer2";
+    private static final String CONSUMER_3 = "consumer3";
+    private static final String CONSUMER_4 = "consumer4";
```

Comment:
on-the-side cleanup... for some reason, IDEA suggests turning this into a constant.